binding/android/Porcupine/porcupine/build.gradle

```diff
@@ -34,3 +33,10 @@ if (file("${rootDir}/publish-mavencentral.gradle").exists()) {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
 }
+
+task copyLibs(type: Copy) {
+    from("${rootDir}/../../lib/android")
+    into("${rootDir}/porcupine/src/main/jniLibs")
+}
+
+preBuild.dependsOn copyLibs
```

Comment:
I think it's time we move to 100% dependency on the gradle copy for our libs and models. Delete all models, built-in keywords and libs from the project, add `.gitkeeps` to the folders, add them to `.gitignore` and finally add the copies to this gradle script. That way we never need to remember to keep them up to date - they just always will be up to date